<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvDecoDialog</class>
 <widget class="QDialog" name="ConvDecoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 50);
border-color: rgb(155, 170, 0);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="QLabel" name="conv_count_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:white;
	font: 12pt &quot;Sitka&quot;;
}
</string>
       </property>
       <property name="text">
        <string>conv count</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="conv_count">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
    font:  9pt &quot;Cascadia Code&quot;;
    color: black;
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                     stop: 0 #F1F1F1, stop: 0.4 #EEEEEE,
                                     stop: 0.5 #E8E8E8, stop: 1.0 #E3E3E3);
    border: 1px solid gray;
    border-radius: 3px;
}

QLineEdit::PlaceHolder{
	color: grey;
}</string>
       </property>
       <property name="text">
        <string>10</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="QLabel" name="max_vert_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:white;
	font: 12pt &quot;Sitka&quot;;
}
</string>
       </property>
       <property name="text">
        <string>max vert count</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="max_vert">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
    font:  9pt &quot;Cascadia Code&quot;;
    color: black;
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                     stop: 0 #F1F1F1, stop: 0.4 #EEEEEE,
                                     stop: 0.5 #E8E8E8, stop: 1.0 #E3E3E3);
    border: 1px solid gray;
    border-radius: 3px;
}

QLineEdit::PlaceHolder{
	color: grey;
}</string>
       </property>
       <property name="text">
        <string>32</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="scale_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:white;
	font: 12pt &quot;Sitka&quot;;
}
</string>
       </property>
       <property name="text">
        <string>scale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Vector3Edit" name="scale">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:white;
	font: 12pt &quot;Sitka&quot;;
}
</string>
       </property>
       <property name="text">
        <string>show mesh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="show_mesh">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="need_log_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	color:white;
	font: 12pt &quot;Sitka&quot;;
}
</string>
       </property>
       <property name="text">
        <string>need log</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="need_log">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin" stdset="0">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="ok_button">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 1px solid white;
border-radius:4px;
border-bottom: 2px solid gray;
border-right: 2px solid gray;
background-color: 
qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, 
			stop:0     rgba(100, 100, 100, 255),
			stop:0.35 rgba(20, 20, 20, 255), 
			stop:0.65 rgba(25, 25, 25, 255),
			stop:1 	  rgba(35, 35, 65, 255));
color: 
qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1,
 stop:0 rgba(175, 175, 200, 255), 
 stop:1 rgba(220, 220, 255, 255));
}
QPushButton::hover{
background-color: 
qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, 
			stop:0     rgba(120, 120, 120, 255),
			stop:0.35 rgba(40, 40, 40, 255), 
			stop:0.65 rgba(45, 45, 45, 255),
			stop:1 	  rgba(55, 55, 85, 255));
color: 
qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1,
 stop:0 rgba(200, 200, 240, 255), 
 stop:1 rgba(235, 245, 255, 255));
border: 1px solid rgb(200, 245, 255);
border-radius:4px;
border-bottom: 1px solid rgb(175, 220, 230);
border-right: 1px solid rgb(175, 220, 230);
}
</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border: 1px solid white;
border-radius:4px;
border-bottom: 2px solid gray;
border-right: 2px solid gray;
background-color: 
qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, 
			stop:0     rgba(100, 100, 100, 255),
			stop:0.35 rgba(20, 20, 20, 255), 
			stop:0.65 rgba(25, 25, 25, 255),
			stop:1 	  rgba(35, 35, 65, 255));
color: 
qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1,
 stop:0 rgba(175, 175, 200, 255), 
 stop:1 rgba(220, 220, 255, 255));
}
QPushButton::hover{
background-color: 
qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, 
			stop:0     rgba(120, 120, 120, 255),
			stop:0.35 rgba(40, 40, 40, 255), 
			stop:0.65 rgba(45, 45, 45, 255),
			stop:1 	  rgba(55, 55, 85, 255));
color: 
qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1,
 stop:0 rgba(200, 200, 240, 255), 
 stop:1 rgba(235, 245, 255, 255));
border: 1px solid rgb(200, 245, 255);
border-radius:4px;
border-bottom: 1px solid rgb(175, 220, 230);
border-right: 1px solid rgb(175, 220, 230);
}
</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Vector3Edit</class>
   <extends>QWidget</extends>
   <header>custom/vector3_edit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ok_button</sender>
   <signal>clicked()</signal>
   <receiver>ConvDecoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>ConvDecoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
